shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/util/TableDatabaseNameJoiner.java

```diff
@@ -0,0 +1,9 @@
+package com.hotels.shunting.yard.replicator.util;
+
+public class TableDatabaseNameJoiner {
```

Comment:
Since this is a util class give it a private constructor

Reply:
Done.